lib/src/main/java/io/cloudquery/scalar/Binary.java

```diff
@@ -37,14 +36,12 @@ public ArrowType dataType() {
     @Override
     public void set(Object value) throws ValidationException {
         if (value == null) {
-            this.valid = false;
             this.value = null;
             return;
         }
 
         if (value instanceof Scalar scalar) {
```

Comment:
I think you can add a `?` to remove some IDE warnings on these instance of checks (other files too)

```suggestion
        if (value instanceof Scalar<?> scalar) {
```

Reply:
done